caldp/process.py

```diff
@@ -113,7 +110,12 @@ def get_output_path(output_uri, ipppssoot):
             output_prefix = test_prefix
         else:
             output_prefix = os.path.join(os.getcwd(), test_prefix)
-    # s3 or astroquery:
+    # s3 - force consistency of paths with caldp-process script
+    elif output_uri.startswith("s3"):
+        if output_uri.endswith("/data"):
+            output_prefix = output_uri
+        else:
+            output_prefix = output_uri + "/data"
     else:
         output_prefix = output_uri
     return output_prefix + "/" + instrument_name + "/" + ipppssoot
```

Comment:
Any reason to not use os.path.join() here?

Reply:
possibly due to the fact that the S3 path won't behave very well with os.path.join()?